full/src/main/java/apoc/uuid/UuidHandler.java

```diff
@@ -163,7 +163,8 @@ public void add(Transaction tx, String label, UuidConfig config) {
             Node node = Util.mergeNode(sysTx, SystemLabels.ApocUuid, null,
                     Pair.of(SystemPropertyKeys.database.name(), db.databaseName()),
                     Pair.of(SystemPropertyKeys.label.name(), label),
-                    Pair.of(SystemPropertyKeys.propertyName.name(), propertyName)
+                    Pair.of(SystemPropertyKeys.propertyName.name(), propertyName),
+                    Pair.of(SystemPropertyKeys.addToSetLabel.name(), config.isAddToSetLabels())
```

Comment:
Honestly, I don't think we should use the config as merge key, but we should use the same approach as here:

https://github.com/neo4j-contrib/neo4j-apoc-procedures/blob/9efa041c0b046d170860ff2fde78e6d1155333a1/core/src/main/java/apoc/trigger/TriggerHandler.java#L123-L132

@JMHReif @fbiville wdyt?

Reply:
That makes sense. Since config might change, it's indeed better not to include it in `mergeNode` and set it afterwards.